packages/ag-charts-enterprise/src/series/sunburst/sunburstSeries.ts

```diff
@@ -312,10 +313,13 @@ export class SunburstSeries extends _ModuleSupport.HierarchySeries<
                     perpendicularWidth =
                         Math.sqrt(outerRadius ** 2 - (perpendicularHeight / 2) ** 2) -
                         labelHeight / (2 * Math.tan(deltaOuterAngle * 0.5));
-                } else {
+                } else if (_Util.normalizeAngle360(deltaInnerAngle) < maxPerpendicularAngle) {
                     // Outer wedge - fit the height to the sector, then fit the width
                     perpendicularHeight = 2 * innerRadius * Math.tan(deltaInnerAngle * 0.5);
                     perpendicularWidth = Math.sqrt(outerRadius ** 2 - (perpendicularHeight / 2) ** 2) - innerRadius;
```

Comment:
The actual root cause was this becoming `NaN`, which made `parallelWidth >= perpendicularWidth` below return false